Solutions/Azure Activity/Analytic Rules/Creation_of_Expensive_Computes_in_Azure.yaml

```diff
@@ -21,27 +20,28 @@ tactics:
 relevantTechniques:
   - T1578
 query: |
-  let tokens = dynamic(["416","208","128","120","96","80","72","64","48","44","40","g5","gs5","g4","gs4","nc12","nc24","nv12"]);
+  let tokens = dynamic(["416","208","192","128","120","96","80","72","64","48","44","40","g5","gs5","g4","gs4","nc12","nc24","nv24"]);
   let operationList = dynamic(["microsoft.compute/virtualmachines/write", "microsoft.resources/deployments/write"]);
   AzureActivity
   | where tolower(OperationNameValue) in (operationList)
-  | where ActivityStatusValue == "Accepted" 
+  | where ActivityStatusValue =~ "Accepted" or ActivityStatusValue =~ "Accept"
```

Comment:
Could we not just do | where ActivityStatusValue startswith "Accept"

Reply:
thanks @petebryan . this is done